src/main/webui/src/app/components/content/common/StoreListingWidget.jsx

```diff
@@ -14,62 +14,96 @@
  * limitations under the License.
  */
 
-import React, {Fragment} from 'react';
-import {Link} from 'react-router-dom';
-import {PropTypes} from 'prop-types';
-import {Utils} from '#utils/AppUtils.js';
+import React, { Fragment, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+import { Utils } from '#utils/AppUtils.js';
```

Comment:
Can we remove these spaces in {}? Seems it violates the es-linter.

Reply:
fixed formating

Reply:
Seems not fixed. Spaces still in {}.